README.md

```diff
@@ -1,162 +1,114 @@
+## Features
 
-The environment is built using [Unity ml-agents](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) **Release 20-stable (Python version 0.30.0)**. **Unity** (game engine) is the engine used to develop the platform.
+**Interdisciplinary Research Platform:**
+- Facilitates research in human, animal, and artificial cognition.
+- Supports cross-disciplinary studies.
+- Enables comparative assessments among humans, animals, and artificial intelligences.
 
-The AnimalAI environment and packages are currently tested on **Windows 11**, **Linux**, and **Mac**, with minimum **Python 3.8.x**, but Python 3.6+ has reported to be working also. **Linux distros** are also working and stable.
+**Comprehensive AI Environment:**
+- Includes a versatile environment for AI experiments, from basic to advanced configurations.
+- Wrap Unity learning environments as a gym(nasium) environment
+- Wrap Unity learning environments as a PettingZoo environment
+- Support for several Deep Reinforcement Learning algorithms (PPO, SAC, MA-POCA, self-play).
 
-**The Unity Project** for the environment is available [here](https://github.com/Kinds-of-Intelligence-CFI/animal-ai-unity-project).
+**Extensive Task Library:**
+- Multiple example tasks.
+- 900 tasks from the Animal-AI Olympics.
+- Procedural Generation functionality
+- Additional tasks for demonstration and experimentation.
 
+**Unity Game Engine:**
+- Utilizes Unity ml-agents.
+- Leverages Unity game engine for advanced simulation capabilities.
+- Fast and robust wrapper.
 
-## Quick Install (please see Release for latest version of AAI-3)
+**Cross-Platform Compatibility:**
+- Compatible with Windows 11, Linux, and MacOS.
+- Supports Python 3.6.x and above.
 
-*see [here](docs/installationGuide.md) for a more detailed installation guide, including information on Python/pip/conda and using the command line during installation*
+**Control Modes:**
+- Player mode for interactive environment control, for human testing.
+- Training mode for Reinforcement Learning, with support for tensorflow analysis.
+- Supports AI model training across different systems.
 
-To get started you will need to:
-1. Clone this repo.
-2. **Install the animalai python package** and requirements by running `pip install animalai` from the root folder.
-3. **Download the latest release of the environment** for your system [here](https://github.com/Kinds-of-Intelligence-CFI/animal-ai/releases):
+**Interactive and Dynamic Environment:**
+- Offers interactive elements for complex AI training.
+- Supports dynamic environment generation.
```

Comment:
nit: I'm not sure what is meant by dynamic here

Reply:
Objects can be dynamic such as rewards moving, spawning objects in the sky, etc... In gaming terms, these constitute dynamism. I just thought its important we tell the user that the environment can be dynamic. I've since added 'dynamic objects' to the end to make it more clear